examples/finance/black_scholes.py

```diff
@@ -0,0 +1,144 @@
+import argparse
+import contextlib
+import time
+
+import cupy
+import numpy
+
+# This sample computes call and put prices for Europian options with
+# Black-Scholes equation. It was based on a sample of the financial package
+# in CUDA toolkit. For details, please see the corresponding whitepaper.
+#
+# The following code shows that CuPy enables us to write algorithms for GPUs
+# without significantly modifying the existing NumPy's code.
+# It also briefly describes how to create your own kernel with CuPy.
+# If you want to speed up the existing code, please define the kernel
+# with cupy.ElementwiseKernel.
+
+
+# Naive implementation of the pricing of options with NumPy.
+def black_scholes(xp, s, x, t, r, v):
```

Comment:
This function is also run with cupy, so it's better to mention it as well.

Reply:
Fixed.